### PR TITLE
Feature: update credentials field for Big Query data source

### PR DESCRIPTION
**Pull request by fredalai**
## Description

 - Send file content (JSON) instead of File to backend verify credentials.

## Issue ticket number

closes #xx

### CODE REVIEW

File: packages/admin-ui/components/pages/setup/dataSources/BigQueryProperties.tsx [hunk 1]
```diff
@@ -4,17 +4,50 @@ import UploadOutlined from '@ant-design/icons/UploadOutlined';
...
       setFileList([uploadFile]);
     }
   };
 
+  return (
+    <Upload
+      accept=".json"
+      fileList={fileList}
+      onChange={onUploadChange}
+      onRemove={() => setFileList([])}
```
Comment (andreashimin) [inline packages/admin-ui/components/pages/setup/dataSources/BigQueryProperties.tsx:42]: The onRemove function also needs to update onChange to the form instance

```ts
const onRemove = () => {
  setFileList([])
  onChange && onChange(null)
}
```

Comment (fredalai) [reply packages/admin-ui/components/pages/setup/dataSources/BigQueryProperties.tsx]: Thanks for the reminder.
I updated the code, remove will make it to the initial state.

**Comment by fredalai** [comment]
Hi @andreashimin,
The code has been changed, please code review if you have available time.

**Review by andreashimin — APPROVED** [review]
LGTM

**Pull request merged**